src/MyEventEmitter.js

```diff
@@ -1,14 +1,102 @@
 'use strict';
 
 class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  constructor() {
+    this.listeners = {};
+  }
+
+  emptyListener(eventName) {
+    if (!this.listeners[eventName]) {
+      this.listeners[eventName] = [];
+    }
+  }
+
+  setValidation(eventName, eventListener) {
+    if (
+      !(typeof eventName === 'string' && typeof eventListener === 'function')
+    ) {
+      throw new Error('ERROR: invalid arguments');
+    }
+  }
```

Comment:
I wouldn't make this methods as a part of class since we're implementing the original class and it doesn't have these methods. lets move them outside a class and use `call` or  `apply` to set `this`